packages/netlify-cms-core/src/components/MediaLibrary/MediaLibraryCreateFolder-old.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
As it's named `-old` I suppose this file should be removed, right?

Reply:
Yes, sorry. I will commit that now.